net/BUILD.gn

```diff
@@ -4224,7 +4224,6 @@ target(gtest_target_type, "net_unittests") {
     "//base:i18n",
     "//base/test:test_support",
     "//base/third_party/dynamic_annotations",
-    "//cobalt/persistent_storage:persistent_settings",
```

Comment:
The only rationale for this change is this line: We _cannot_ have reverse dependencies.

Reply:
yaay! thanks! this helps make the unresolved symbol resolution much easier